charts/openstad-headless/templates/admin/deployment.yaml

```diff
@@ -68,6 +68,16 @@ spec:
               secretKeyRef:
                 key: rateWindowMs
                 name: {{ template "openstad.ratelimit.secret.fullname" . }}
+          - name: NEXT_PUBLIC_HASH_IP_ADDRESSES
+            valueFrom:
+              secretKeyRef:
+                key: hashIpAddresses
+                name: {{ template "openstad.ipaddress.secret.fullname" . }}
+          - name: HASH_IP_SALT
+            valueFrom:
+              secretKeyRef:
+                key: hashIpSalt
+                name: {{ template "openstad.ipaddress.secret.fullname" . }}
```

Comment:
Why does the admin need to have the salt? And shouldn't it be prefixed with `NEXT_PUBLIC_` in that case?